### DISCUSSION TITLE
No available formula with the name "dwgsim".

### DISCUSSION THREAD

**Discussion by yaseminsucu** [biostars/biostar-handbook-issues#221] [Q&A]
I got an error while installing DWGSIM with the **brew install dwgsim** command. Is this tool still exist? Thanks!

**Comment by ialbert** [comment]
I have recently written a wrapper over `wgsim` that does all that `dwgsim` does

https://github.com/ialbert/pywgsim

I need to modify the chapter on `dwgsim` and recommend `pywgsim` instead.

**Reply by yaseminsucu** [reply]
Thanks so much for your prompt response and update!
